file-formats/ints/examples/z_aff_example_ints.ints.json

```
@@ -0,0 +1,27 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "ISLM ADT for INTS - Scenario",
+    "originalLanguage": "en"
+  },
+  "generalInformation": {
+    "scenarioType": "SAPGENAI",
+    "scenarioTechnology": "SIDEBYSIDE",
```

If you follow my proposal, also the example needs to be updated

```suggestion
    "scenarioTechnology": "sideBySide",
```

---

Hi @schneidermic0 ,

Actually, we have to keep this SIDEBYSIDE value instead of sideBySide. This is because, we are validating this value in lot of places in our application code. It will have huge impact for our existing code and object those were created already.

So, we want to keep this value as SIDEBYSIDE.

Thanks,
Raghav

---

IF_ISLM2_CONSTANTS=>GC_TECH_SIDE_BY_SIDE - This is used in lot of places for validation. This is the reason to keep the value SIDEBYSIDE as it is.

---

In your ABAP code it is still "SIDEBYSIDE" only the JSON representation will change to "sideBySide". This conversion is done by the transformation. So the values in the database are sill the same as before and as stated in the value statement of your constants.

---

Hi @schneidermic0 ,

Even though I change the enum constant to "sideBySide", our GET_CONTENT method is giving this value from our DB only. So, JSON will anyway have "SIDEBYSIDE".

Do you want to change GET_CONTENT code, to display the "sideBySide" in the editor and JSON output?

If yes, we need to put IF condition in SAVE_ADDITIONAL_PROPERTIES when saving this data, to convert from "sideBySide" to "SIDEBYSIDE" and another IF condition when during the GET_CONTENT to convert from "SIDEBYSIDE" to "sideBySide".

Please let me know your comments.

Thanks,
Raghav

---

Hi @schneidermic0 ,

As discussed, fixed the above comments and pushed the changes to GitHub.
Please review and let me know your comments.

Thanks,
Raghav